### PR TITLE
New single message

### LINKED ISSUE #236: New Single Message
Screen Shot 2020-07-14 at 16 12 00

### PR DESCRIPTION
**Pull request by LouiaR**
Fixes #236

### CODE REVIEW

File: src/components/Molecules/SingleMessageDS/SingleMessageDs.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        Heading Line 1 Heading Line 2
+      </Title>
+      <Text tag='p'>
+        Whatever you’ve got planned, the Sport Relief shop has everything you need
+        to get you looking your best while you’re raising some cash. Also
+        available in Sainsbury’s stores and online and in selected Argos stores.
+      </Text>
+    </SingleMessageDs>
+</div>;
+```
```
Comment (AndyEPhipps) [inline src/components/Molecules/SingleMessageDS/SingleMessageDs.md:95]: @LouiaR, could we have examples of non-CTA ones too? Just so we're covering a few more examples to make sure there's no issues.

**Comment by AndyEPhipps** [comment]
Screen Shot 2020-07-27 at 14 44 32
I'm not sure how far we're supporting, but this is iPhone 7, iOS 10.. just in case it's an easy fix

File: src/components/Molecules/SingleMessageDS/SingleMessage.test.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import 'jest-styled-components';
+import renderWithTheme from '../../../hoc/shallowWithTheme';
+import SingleMessageDs from './SingleMessageDs';
+import Text from '../../Atoms/Text/Text';
+import data from '../../../styleguide/data/data';
+
+it('renders correctly', () => {
+  const tree = renderWithTheme(
+    <SingleMessageDs
```
[the remark this answers is not in the document]
Comment (LouiaR) [reply src/components/Molecules/SingleMessageDS/SingleMessage.test.js:10]: We have still have the previus a single message

**Comment by LouiaR** [comment]
> Screen Shot 2020-07-27 at 14 44 32
> I'm not sure how far we're supporting, but this is iPhone 7, iOS 10.. just in case it's an easy fix

Try to open the component separately.

**Comment by AndyEPhipps** [comment]
> Try to open the component separately.

It looks the same in your Contentful PR test env unfortunately :/

**Comment by AndyEPhipps** [comment]
Screen Shot 2020-07-27 at 16 20 15
👌

File: src/components/Molecules/SingleMessageDS/SingleMessage.test.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (AndyEPhipps) [reply src/components/Molecules/SingleMessageDS/SingleMessage.test.js:10]: haha I really meant what does Ds mean

**Pull request merged**